main.go

```diff
@@ -180,6 +180,7 @@ func setup() {
 	viper.SetDefault(devEnv, false)
 
 	// db
+	viper.SetDefault(dbDSNEnv, "postgres://admindbuser:admindbpassword@localhost:5432/admindb?sslmode=disable")
```

Comment:
Setting a default will make it so that we always pick this. We should leave it out to be empty by default.

Reply:
This would be the case in `initialize/main.go` as well.

Reply:
Fixed.